src/platform/nxp/k32w/k32w0/BUILD.gn

```diff
@@ -46,7 +46,6 @@ static_library("nxp_platform") {
     "K32W0Config.h",
     "KeyValueStoreManagerImpl.cpp",
     "KeyValueStoreManagerImpl.h",
-    "Logging.cpp",
```

Comment:
Removed but not added anywhere?

Reply:
Also added here:
https://github.com/project-chip/connectedhomeip/blob/b0a105817784e5bd48ad3674cefaf64294936480/src/platform/nxp/BUILD.gn#L23-L34